exporter/chronicleexporter/grpc_exporter.go

```diff
@@ -129,15 +129,11 @@ func (exp *grpcExporter) ConsumeLogs(ctx context.Context, ld plog.Logs) error {
 	if err != nil {
 		return fmt.Errorf("marshal logs: %w", err)
 	}
-	successfulPayloads := []*api.BatchCreateLogsRequest{}
 	for _, payload := range payloads {
 		if err := exp.uploadToChronicle(ctx, payload); err != nil {
-			// If there is an error only report
-			// the bytes successfully sent
-			exp.countAndReportBatchBytes(ctx, successfulPayloads)
+			// Don't count bytes on failure - retry will count them on success
 			return err
 		}
-		successfulPayloads = append(successfulPayloads, payload)
```

Comment:
I think we still want to do this on the cases where retry on failure is not enabled. This seems like a good solution otherwise.